### PR TITLE
[WIP] Documentation for the gRPC module

### PR DESCRIPTION
**Pull request by rogchap**
docs for: https://github.com/loadimpact/k6/pull/1623

### CODE REVIEW

File: src/data/markdown/docs/02 javascript api/09 k6-grpc/20 Client/10-Client-load-importpaths-protoFiles.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+|------|-------------|
+| Array<[MethodInfo](#methodinfo)> | List of service RPC methods that where successfully parsed. |
+
+## MethodInfo
+
+*MethodInfo* is the information on the available RPC method.
+
+| Name | Type | Description |
+|------|------|-------------|
+| `MethodInfo.fullMethod` | string | The full RPC method string, i.e., `package.service/method`. |
```
Comment (amossc) [inline src/data/markdown/docs/02 javascript api/09 k6-grpc/20 Client/10-Client-load-importpaths-protoFiles.md:27]: `s/i.e./e.g./`

ref: https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/

File: src/data/markdown/docs/02 javascript api/09 k6-grpc.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+| [Client.load(importPaths, ...protoFiles)](/javascript-api/k6-grpc/client/client-load-importpaths----protofiles) | Loads and parses the given protocol buffer definitions to be made available for RPC requests. |
+| [Client.connect(address [,params])](/javascript-api/k6-grpc/client/client-connect-address-params) | Connects to a given gRPC service. |
+| [Client.invokeRPC(url, request [,params])](/javascript-api/k6-grpc/client/client-invokerpc-url-request-params) | Makes a unary RPC for the given service/method and returns a [Response](/javascript-api/k6-grpc/response). |
+| [Client.close()]() | Close the connection to the gRPC service. |
+| [Params](/javascript-api/k6-grpc/params) | RPC Request specific options. |
+| [Response](/javascript-api/k6-grpc/response) | Returned by RPC requests. |
+
+| Constant | Description |
+|----------|-------------|
+| `StatusOK` | OK is returned on success. |
```
Comment (amossc) [inline src/data/markdown/docs/02 javascript api/09 k6-grpc.md:24]: Do you have to repeat the name of the status on the second column? It looks redundant.

File: src/data/markdown/docs/02 javascript api/09 k6-grpc/20 Client/30-Client-invokerpc-url-request-params.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Client.invokeRPC(url, request [,params])"
+---
+
+Invokes an unary RPC request to the given method.
+
+The given method to invoke must have it's RPC schema previously loaded via the [Client.load()](/javascript-api/k6-grpc/client/client-load-importpaths----protofiles) function, otherwise an
```
Comment (amossc) [inline src/data/markdown/docs/02 javascript api/09 k6-grpc/20 Client/30-Client-invokerpc-url-request-params.md:7]: `s/it's/its/`

File: src/data/markdown/docs/02 javascript api/09 k6-grpc/20-Client.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+---
+
+`Client` is a gRPC client that can interact with a gRPC server.
+
+>  ⚠️ **Note**: Only unary RPCs are currently supported, ie. there is no support for client, server or bidirectional streaming. 
+
+| Method | Description |
+|--------|-------------|
+| [Client.load(importPaths, ...protoFiles)](/javascript-api/k6-grpc/client/client-load-importpaths----protofiles) | Loads and parses the given protocol buffer definitions to be made available for RPC requests. |
+| [Client.connect(address [,params])](/javascript-api/k6-grpc/client/client-connect-address-params) | Open's a connection to the given gRPC server. |
```
Comment (amossc) [inline src/data/markdown/docs/02 javascript api/09 k6-grpc/20-Client.md:12]: `s/Open's/Opens/`

File: src/data/markdown/docs/02 javascript api/09 k6-grpc/20-Client.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+| [Client.invokeRPC(url, request [,params])](/javascript-api/k6-grpc/client/client-invokerpc-url-request-params) | Makes a unary RPC for the given service/method and returns a [Response](/javascript-api/k6-grpc/response). |
```
Comment (amossc) [inline src/data/markdown/docs/02 javascript api/09 k6-grpc/20-Client.md:13]: `an unary`: https://www.a-or-an.com/a_an/unary

**Review by amossc — CHANGES_REQUESTED** [review]

**Comment by simskij** [comment]
This was added in #144. Closing without merging

**Pull request closed without merging**